src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,138 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is a commonly used approach. This approach, which is also known as 'Big Bang' (and by other names), refers to the process where your data is moved in its entirety from a source system to a target system within a defined period of time. This can involve some application downtime or service degradation.
```

Comment:
Updated, thanks!  

(PS I appreciate the suggestion via the Github UI tool, just wanted to let you know I'm adding them manually. I don't accept them via the web interface since I rebase locally and it's hard to manage keeping my local edits in sync with the weird commits the UI makes. Just in case it seems strange that I'm saying "yes thanks" but not clicking the button here)